playbooks/roles/mysql/tasks/main.yml

```diff
@@ -21,4 +21,12 @@
   when: remove_experimental_mysql
 
 - include: mysql.yml
-  when: (mysql_56_installed.rc == 1) or (remove_experimental_mysql)
\ No newline at end of file
+  when: (mysql_56_installed.rc == 1) or (remove_experimental_mysql)
+
+- name: Create remote root user
```

Comment:
@melvinsoft The `mysql_init` role needs to be run from the Insights playbook, otherwise the Analytics databases and users won't be created. There's also a [task](https://github.com/appsembler/configuration/blob/appsembler/eucalyptus/develop/playbooks/roles/analytics_pipeline/tasks/main.yml#L187-L195) in the `analytics_pipeline` role that manipulates the database. (That task really shouldn't be in that role.. it should be in `edxlocal`/`mysql_init` instead.)

I see that [your playbook](https://github.com/appsembler/configuration/commit/2e25ceca953e22c0a9d2b9ce2f39b409743c8cde#diff-c9150f389ab54e468ddaeb4790caa2aa) uses a local MySQL. For that case, a remote root user isn't needed, but I thought we decided to store everything in a single database..?